ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -16,15 +16,19 @@
 
 import static org.junit.Assert.assertTrue;
 
-@Ignore
+
 public class LoginTest extends ReferenceApplicationTestBase {
     private HomePage homePage;
+    
+    public void initiateHomePage() {
+        homePage = new HomePage(page);
+        assertPage(homePage.waitForPage());
+    }
 
     @Test
-    @Category(org.openmrs.reference.groups.BuildTests.class)
+    @Category(BuildTests.class)
     public void verifyModulesAvailableOnHomePage() throws Exception {
```

Comment:
Remove the `throws` here and also from those respective methods that do not need it